src/Workspaces/CSharp/Portable/Classification/SyntaxClassification/NameSyntaxClassifier.cs

```diff
@@ -116,12 +116,24 @@ internal class NameSyntaxClassifier : AbstractSyntaxClassifier
             Text.TextSpan span,
             ArrayBuilder<ClassifiedSpan> result)
         {
-            if (symbol?.IsStatic == true
-                && (!symbol.IsKind(SymbolKind.Field) || symbol.ContainingType?.IsEnumType() == false) // TODO: Since Enum members are always static is it useful to classify them as static?
-                && !symbol.IsKind(SymbolKind.Namespace)) // TODO: Since Namespace are always static is it useful to classify them as static?
+            if (symbol is null || !symbol.IsStatic)
             {
-                result.Add(new ClassifiedSpan(span, ClassificationTypeNames.StaticSymbol));
+                return;
             }
+
+            var isEnumMember = symbol.IsKind(SymbolKind.Field) && symbol.ContainingType?.IsEnumType() == true;
+            if (isEnumMember) // TODO: Since Enum members are always static is it useful to classify them as static?
+            {
+                return;
+            }
+
+            var isNamespace = symbol.IsKind(SymbolKind.Namespace);
+            if (isNamespace) // TODO: Since Namespaces are always static is it useful to classify them as static?
```

Comment:
namespaces are static?

Reply:
That is what the symbol returns. Surprised me too =)

Reply:
My expectation as a user would be that the 'static' classification for C# would refer to members that were declared with the 'static' modifier.  VB would be similar, except with the understanding that module-members are all implicitly 'shared'.  

Importantly, this modifier only matters for symbols thta could ever be *non*-static.  as the modifier/classification is used to distinguish between the two.  Since there are no 'instance namespaces' or 'instance enum members', i would not classify them as 'static'.

Reply:
That seems to be the consensus opinion.